ArticleTemplates/articleTemplateImmersiveHeight.html

```diff
@@ -1 +1 @@
-height: calc(100vh - __BODY_OFFSET_TOP__px);
\ No newline at end of file
+height: calc(90vh - __BODY_OFFSET_TOP__px);
```

Comment:
iOS substitutes this value when rendering the template (so does android, i think, but android uses more specific styling that ends up overriding this)

ref: https://github.com/guardian/ios-live/blob/master/GLA/GLA/Classes/GLAArticleHtmlRenderer.m#L1476